map.html

```diff
@@ -1,4 +1,14 @@
 ---
 layout: default
 ---
+<div class="dropdown toggle">
+  <span class="dropdown__title">Layer</span>
+  <div class="dropdown__content">
```

Comment:
This is a bit nitpicky, but I would consider switching to the more semantic `select` and `option` tags for the dropdown if doing so doesn't require a complete overhaul of the CSS.